tests/test_processing_common.py

```diff
@@ -126,13 +129,12 @@ def test_tokenizer_defaults_preserved_by_kwargs(self):
         if "image_processor" not in self.processor_class.attributes:
             self.skipTest(f"image_processor attribute not present in {self.processor_class}")
         image_processor = self.get_component("image_processor")
-        tokenizer = self.get_component("tokenizer", max_length=117)
+        tokenizer = self.get_component("tokenizer", max_length=117, padding="max_length")
```

Comment:
Does this mean the padding behaviour of the processors isn't backwards compatible i.e. it used to default to `"max_length"` and doesn't anymore? Or is this because of the new processors being tested?

Reply:
the latter, this is just because of new processors being tested - no weird BC behaviour here as far as I know